src/components/home/Home.js

```diff
@@ -83,7 +83,7 @@ function Home() {
       (collection) => collection.creator === NEXT_PUBLIC_GRAASPER_ID,
     );
 
-    if (collectionsGraasper.isEmpty()) {
+    if (collectionsGraasper?.isEmpty()) {
```

Comment:
If `collectionsGraasper` in `undefined` it won't display null, will it be the correct behavior?